packages/workbox-cache-expiration/CacheExpirationManager.mjs

```diff
@@ -149,12 +158,14 @@ class CacheExpirationManager {
   }
 
   /**
-   * @return {Promise}
+   * @param {Array<string>} urls Array of URLs to delete from IDB
    *
    * @private
    */
-  _deleteFromIDB() {
-    return Promise.resolve();
+  async _deleteFromIDB(urls) {
+    for (let url of urls) {
```

Comment:
Nit: You can use `const` instead of `let` here. (Which, I found that confusing when I realized that...)